src/object-inspector/ObjectInspector.js

```diff
@@ -12,8 +12,10 @@ const createIterator = (showNonenumerable, sortObjectKeys) => {
     const shouldIterate = (typeof data === 'object' && data !== null) || typeof data === 'function';
     if (!shouldIterate) return;
 
+    const isArray = Array.isArray(data);
```

Comment:
rename this to `dataIsArray`? we are using another isArray just a few lines down, might be confusing here